push_notification_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
+	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
Out of curiosity, why is `jwt` needed?

Reply:
It's not in this case, as the package that lives there is called `jwt`.

Some lesser editors just check if the `folder name == package name`, and then suggestion/automatically add the alias.